### PR TITLE
[#SC-3949] Fixed issue with displaying team members

### PR DESCRIPTION
**Pull request by qnowak**
# Description
Members list was not displayed.

schuler

## Links to Tickets or other pull requests

- https://ticketsystem.schul-cloud.org/browse/SC-3949

### CODE REVIEW

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

File: controllers/teams.js [hunk 1]
```diff
@@ -17,6 +18,20 @@ moment.locale('de');
...
+	const stringifyValue = JSON.stringify(jsonValue);
+	const escapedValueList = ['display: none', stringifyValue];
+	escapedKeyList.forEach((entity, i) => {
+		const escapedKey = handlebars.escapeExpression(entity);
+		const escapedValue = handlebars.escapeExpression(escapedValueList[i]);
+		attributes.push(`${escapedKey}="${escapedValue}"`);
+	});
+	const escapedOutput = `<td ${attributes.join(' ')} />`;
+	return new handlebars.SafeString(escapedOutput);
+});
```
Comment (CordlessWool) [inline controllers/teams.js:33]: shouldn't be done as an helper. Please do this directly in handlebars. You can write a handlebar component if noone exists. Also Stringify is a to regular name to do something with tables

**Comment by seulki-lee-cloud** [comment]
1. Could you please make a space style (--space-lg) for each space? I marked with pink square 
2. Could you please make all text in same line? I marked with red line

Thanks so much :D

Screenshot 2020-05-11 at 12 01 20

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Comment by malcm** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 50
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/schul-cloud/schulcloud-client/pullRequest?prid=5479169&bid=17956863)

**Pull request merged**